packages/dev/core/src/Materials/Textures/textureCreationOptions.ts

```diff
@@ -1,3 +1,5 @@
+import type { InternalTexture } from ".";

```

Comment:
I don't know if it's important, but shouldn't it be `import type { InternalTexture } from "./internalTexture";`?

Reply:
lint caught this also